src/main/resources/openapi/transcriptions.yaml

```diff
@@ -0,0 +1,60 @@
+openapi: 3.0.1
+
+servers:
+  - url: http://localhost:4550/
+
+info:
+  description: Modernised DARTS (Digital Audio Recording and Transcription Service).
+  version: "0.0.1"
+  title: Modernised DARTS
+
+paths:
+  /transcriptions:
+    post:
+      tags:
+        - Transcription
+      summary: Users can request transcription
+      operationId: requestTranscription
+      description: Adds a user transcription request for processing
+      responses:
```

Comment:
I think you will need to add some error response examples here for the case_id / hearing_id not found (404) api errors?

Reply:
The ticket states : Error handling scenarios will be handled in [DMP-768](https://tools.hmcts.net/jira/browse/DMP-768) so I figured that should be part of that ticket?